lib/utils/log/slog_handler.go

```diff
@@ -536,7 +536,14 @@ func NewSlogJSONHandler(w io.Writer, cfg SlogJSONHandlerConfig) *SlogJSONHandler
 						return slog.Attr{}
 					}
 
-					file, line := getCaller(a)
+					file := "UNKNOWN"
+					var line int
+					// The slog.NewJSONHandler will inject "source" Attr when AddSource is true.
```

Comment:
Should we avoid the "source" key in application code? @rosstimothy

Reply:
Yeah I was just checking sloglint's config and repos to see if I found exactly that.